tests/spec/core/best-practices-spec.js

```diff
@@ -21,8 +21,8 @@ describe("Core — Best Practices", () => {
     const doc = await makeRSDoc(ops);
     const pls = doc.body.querySelectorAll("span.practicelab");
     const bps = doc.getElementById("bp-summary");
-    expect(pls[0].textContent).toEqual("Best Practice 1: BP1");
-    expect(pls[1].textContent).toEqual("Best Practice 2: BP2");
+    expect(pls[0].textContent).toEqual("Best Practice 1:BP1");
```

Comment:
We probably want a space after the :

Reply:
The space is there if you look in the example, but because of how I have styled it and split the title if I put a space there then the underline of the title goes out too far.  I can change that though if needed.

Reply:
The problem is if someone copy/pastes the header, the space won’t be there. Thus,  I’d prefer the space.

Reply:
ah ok, I understand! Have fixed

Reply:
Just a nit, always use toBe(). To equal works, but it means something slightly different.